x/gamm/types/constants.go

```diff
@@ -12,6 +12,8 @@ const (
 	// Raise 10 to the power of SigFigsExponent to determine number of significant figures.
 	// i.e. SigFigExponent = 8 is 10^8 which is 100000000. This gives 8 significant figures.
 	SigFigsExponent = 8
+	// TODO: Current fixed cost gas fee per swap -- turn this into a param in the future.
+	GasFeeForSwap = 10000
```

Comment:
Is there a reason to not directly just make this a param?

Reply:
We decided to make it a fixed cost for now to avoid  needing to set up migration code for a change this small: https://github.com/osmosis-labs/osmosis/issues/1903#issuecomment-1173907587